plugin/trino-lancedb/pom.xml

```diff
@@ -0,0 +1,417 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>446-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-lancedb</artifactId>
+    <packaging>trino-plugin</packaging>
+    <description>Trino - LanceDB connector</description>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.lancedb</groupId>
+            <artifactId>lance-core</artifactId>
+            <version>0.1-SNAPSHOT</version>
```

Comment:
For the merge this will have to be an actual release available in Maven Central .. I can help with getting that going

Reply:
Looks like you can switch to 0.0.1 now .. also introduce a property dep.lancedb.version and use it in both dependencies

Reply:
yeah i released it with just the osx binary. i think we need the ubuntu release version. will prep another one but should be quick

Reply:
@walterddr Should the [lance-core](https://mvnrepository.com/artifact/com.lancedb/lance-core) and [lancedb-core](https://mvnrepository.com/artifact/com.lancedb/lancedb-core) dependencies use the same version number? Currently, the lance-core dependency uses version 0.0.5 even though the latest version is 0.18.0. And the lancedb-core dependency uses version 0.0.3.

Reply:
this is fine. they are not release on the same cadence IIUC